src/components/Operatives/OperativeList.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types'
+import Link from 'next/link'
+
+const OperativeList = ({ operatives, workOrderReference }) => {
+  return (
+    <>
+      <h4 className="lbh-heading-h4">Operatives</h4>
+      <ol className="govuk-list govuk-!-margin-top-6 govuk-!-margin-bottom-9">
+        {operatives.map((operative, index) => {
+          const percentageDisplay = operative.jobPercentage
+            ? `${operative.jobPercentage}%`
+            : 'N/A'
+          return process.env.NEXT_PUBLIC_OPERATIVE_MANAGEMENT_MOBILE_ENABLED ===
```

Comment:
Not for this PR but we should probably look at some kind of feature flag npm package